org.eclipse.winery.topologymodeler.ui/src/app/node/node.component.ts

```diff
@@ -21,20 +29,20 @@ import { WineryActions } from '../redux/actions/winery.actions';
   templateUrl: './node.component.html',
   styleUrls: ['./node.component.css'],
 })
-export class NodeComponent implements OnInit, AfterViewInit, OnChanges {
+export class NodeComponent implements OnInit, AfterViewInit {
   public items: string[] = ['Item 1', 'Item 2', 'Item 3'];
   public accordionGroupPanel = 'accordionGroupPanel';
   public customClass = 'customClass';
-  connectorEndpointVisible;
+  connectorEndpointVisible = false;
   startTime;
   endTime;
-  @Input() needsToBeFlashed: boolean;
   longpress = false;
   makeSelectionVisible = false;
+  setFlash = false;
+  @Input() needsToBeFlashed: boolean;
   @Input() title: string;
   @Input() name: string;
-  @Input() left: number;
-  @Input() top: number;
+  @Input() otherAttributes: any;
```

Comment:
it would be easier if you just have one input. One object with all the desired properties you need.
Would be more readable and shortens the code